contracts/test/ProviderRegistryTest.sol

```diff
@@ -81,30 +86,36 @@ contract ProviderRegistryTest is Test {
         vm.deal(address(this), 100 ether);
     }
 
-    function test_VerifyInitialContractState() public view {
+    function testVerifyInitialContractState() public view {
         assertEq(providerRegistry.minStake(), 1e18 wei);
-        assertEq(providerRegistry.feeRecipient(), feeRecipient);
+        assertEq(feePercent, feePercent);
+        assertEq(withdrawalDelay, withdrawalDelay);
         assertEq(providerRegistry.feePercent(), feePercent);
         assertEq(providerRegistry.preConfirmationsContract(), address(0));
         assertEq(providerRegistry.providerRegistered(provider), false);
+        (address recipient, uint256 accumulatedAmount, uint256 lastPayoutBlock, uint256 payoutPeriodBlocks) = bidderRegistry.protocolFeeTracker();
+        assertEq(recipient, feeRecipient);
+        assertEq(payoutPeriodBlocks, protocolFeePayoutPeriodBlocks);
+        assertEq(lastPayoutBlock, block.number);
+        assertEq(accumulatedAmount, 0);
     }
 
-    function testFail_ProviderStakeAndRegisterMinStake() public {
+    function testFailProviderStakeAndRegisterMinStake() public {
```

Comment:
What is the purpose of renaming test functions? It seems like Foundry is using the same notation that we have:
https://book.getfoundry.sh/forge/writing-tests

Reply:
Here is the list from their documentation for the allowed test function names:

https://github.com/ScopeLift/scopelint/blob/1857e3940bfe92ac5a136827374f4b27ff083971/src/check/validators/test_names.rs#L106-L143

Reply:
These were originally changed because the vs code linter integration was failing to recognize our `.solhintignore` file, and the test files were filled with naming errors. 

imo we should stay consistent to the linter we chose, solhint. Foundry doesn't enforce we use scopelint or any linter at all. Futher on in the first link you've sent, the foundry docs use the naming convention that scopelint would call bad:
```solidity
contract ContractTest is Test {
    uint256 a;
    uint256 b;

    function beforeTestSetup(
        bytes4 testSelector
    ) public pure returns (bytes[] memory beforeTestCalldata) {
        if (testSelector == this.testC.selector) {
            beforeTestCalldata = new bytes[](2);
            beforeTestCalldata[0] = abi.encodePacked(this.testA.selector);
            beforeTestCalldata[1] = abi.encodeWithSignature("setB(uint256)", 1);
        }
    }

    function testA() public {
        require(a == 0);
        a += 1;
    }

    function setB(uint256 value) public {
        b = value;
    }

    function testC() public {
        assertEq(a, 1);
        assertEq(b, 1);
    }
}

```

Reply:
I was wrong, solhint does have a rule that is consistent with what you're saying: https://protofire.github.io/solhint/docs/rules/naming/foundry-test-functions.html. For this rule tho, looks like we'd need to create a separate `.solhint.json` for the testing folder

Reply:
Sorry, I was referring to this page, there they are more explicit:
https://book.getfoundry.sh/tutorials/best-practices#tests

Reply:
Fixed in https://github.com/primev/mev-commit/commit/33d4e1e05a101473820367a1e3e47b6f34678a44